schema/mysql/v8/visibility/schema.sql

```diff
@@ -45,6 +45,9 @@ CREATE TABLE executions_visibility (
   TemporalNamespaceDivision     VARCHAR(255)  GENERATED ALWAYS AS (search_attributes->>"$.TemporalNamespaceDivision"),
   BuildIds                      JSON          GENERATED ALWAYS AS (search_attributes->"$.BuildIds"),
   TemporalPauseInfo            JSON          GENERATED ALWAYS AS (search_attributes->"$.TemporalPauseInfo"),
+  TemporalWorkerDeploymentVersion    JSON    GENERATED ALWAYS AS (search_attributes->"$.TemporalWorkerDeploymentVersion"),
+  TemporalWorkflowVersioningBehavior JSON    GENERATED ALWAYS AS (search_attributes->"$.TemporalWorkflowVersioningBehavior"),
+  TemporalWorkerDeployment           JSON    GENERATED ALWAYS AS (search_attributes->"$.TemporalWorkerDeployment"),
```

Comment:
This definition is for KeywordList. For Keyword, follow the example for `TemporalNamespaceDivision`.